### DISCUSSION TITLE
Druva went live with ASC

### DISCUSSION THREAD

**Discussion by DigiLiviu** [Adobe-Marketing-Cloud/asset-share-commons#1068] [Show and tell]
Thanks so much @davidjgonzalez for all your support on these topics and discussions!

We successfully took Asset Share Commons live with our client Druva - https://www.druva.com/resources

**Comment by davidjgonzalez** [comment]
👍 👏 🥳 🎈 -- this is awesome @DigiLiviu - Congrats! Thanks so much for engaging with the project, providing feedback, asking questions, etc.! Its the only way we can get better! 🙇
